### PR TITLE
TSC meeting notes 08-05-2019

### PR DESCRIPTION
**Pull request by michdolan**

### CODE REVIEW

File: docs/tsc/meetings/2019-08-05.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    - Doug: Could put a license file in the test data directory which applies 
+      to all files.
+    - John: Agrees that would be a good approach. BSD-3-Clause is the most 
+      appropriate license for these files.
+    - **TODO**: Michael will add LICENSE.md file to test/data dir in PR #783
+    - **TODO**: Larry will discuss the flagged “Cloudy with a Chance of 
+      Meatballs” images with Sony.
+    - Kevin: We also need to update the LICENSE on the OCIO website
+    - Michael: That is pulled from GitHub, but currently from the RB-1.1 
+      branch. since the license cleanup work is being done in master, is it Ok
```
Comment (KevinJW) [inline docs/tsc/meetings/2019-08-05.md]: Nit picks: since -> Since

File: docs/tsc/meetings/2019-08-05.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      appropriate license for these files.
+    - **TODO**: Michael will add LICENSE.md file to test/data dir in PR #783
+    - **TODO**: Larry will discuss the flagged “Cloudy with a Chance of 
+      Meatballs” images with Sony.
+    - Kevin: We also need to update the LICENSE on the OCIO website
+    - Michael: That is pulled from GitHub, but currently from the RB-1.1 
+      branch. since the license cleanup work is being done in master, is it Ok 
+      to wait for a 2.0 release next year to update the site?
+    - John: That should be Ok.
+    - Kevin: Let’s make a note to not forget it.
```
Comment (KevinJW) [inline docs/tsc/meetings/2019-08-05.md]: Should this be an Issue?

File: docs/tsc/meetings/2019-08-05.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    - John: There’s often a good reason why people have not been around.
+    - Patrick: We could commit the current COMMITTERS file as-is, and contact 
+      inactive committers to inquire of interest in continued participation.
+    - **TODO**: Michael will reach out to those with admin/write status to ask 
+      if they would like to remain involved in OCIO.
+
+* SIGGRAPH Recap
+    - Doug: Good turnout to BoF, in spite of it happening at the same time as 
+      USD BoF.
+        - Side-conversations and feeback on the event were positive.
```
Comment (KevinJW) [inline docs/tsc/meetings/2019-08-05.md]: feeback -> feedback

File: docs/tsc/meetings/2019-08-05.md (hunk 2 above)
Comment (michdolan) [reply docs/tsc/meetings/2019-08-05.md]: #790 - more of a general reminder, but inherently accounts for the License page.

**Pull request merged**